<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageVersionForm</class>
 <widget class="QWidget" name="PackageVersionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>493</width>
        <height>820</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Icon:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelIcon">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditTitle</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditTitle">
         <property name="toolTip">
          <string>package title</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Version:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditVersion</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditVersion">
         <property name="toolTip">
          <string>version number</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Description:</string>
         </property>
         <property name="buddy">
          <cstring>textEditDescription</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="textEditDescription">
         <property name="toolTip">
          <string>package description</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>License:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelLicense">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>license</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Installation path:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPath</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditPath">
         <property name="toolTip">
          <string>installation path on disk or empty if this package version is not installed</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Detection info:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditDetectionInfo</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditDetectionInfo">
         <property name="toolTip">
          <string>detection info or empty</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Internal name:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditInternalName</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEditInternalName">
         <property name="toolTip">
          <string>internal package name</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditStatus</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEditStatus">
         <property name="toolTip">
          <string>status of this package version</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Download URL:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="labelDownloadURL">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
          <string>direct download link for the package binary</string>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://www.test.de&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;TextLabel&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditType</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="lineEditType">
         <property name="toolTip">
          <string>one-file or zip</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Hash sum:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditSHA1</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="lineEditSHA1">
         <property name="toolTip">
          <string>cryptographic hash sum of the binary (either SHA-1 or SHA-256)</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Important files:</string>
         </property>
         <property name="buddy">
          <cstring>textEditImportantFiles</cstring>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QTextEdit" name="textEditImportantFiles">
         <property name="toolTip">
          <string>an entry in the start menu will be created for each important file mentioned here</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Dependencies:</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Text files:</string>
         </property>
         <property name="buddy">
          <cstring>tabWidgetTextFiles</cstring>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QTabWidget" name="tabWidgetTextFiles">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="toolTip">
          <string>text files with their contents that will be created in the package directory additionally to the files provided by the downloaded binary</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QFrame" name="frameDependencies">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Command line tools:</string>
         </property>
         <property name="buddy">
          <cstring>textEditCmdFiles</cstring>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QTextEdit" name="textEditCmdFiles">
         <property name="toolTip">
          <string>an entry in the directory %allusersprofile%NpackdCommands will be created for each command line tool mentioned here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>lineEditVersion</tabstop>
  <tabstop>textEditDescription</tabstop>
  <tabstop>labelLicense</tabstop>
  <tabstop>lineEditPath</tabstop>
  <tabstop>lineEditDetectionInfo</tabstop>
  <tabstop>lineEditInternalName</tabstop>
  <tabstop>lineEditStatus</tabstop>
  <tabstop>labelDownloadURL</tabstop>
  <tabstop>lineEditType</tabstop>
  <tabstop>lineEditSHA1</tabstop>
  <tabstop>textEditImportantFiles</tabstop>
  <tabstop>tabWidgetTextFiles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
